src/utils/update/fetch.managing.officers.private.data.ts

```diff
@@ -32,16 +33,14 @@ export const fetchManagingOfficersPrivateData = async (appData: ApplicationData,
 };
 
 const mapManagingOfficersPrivateData = (moPrivateData, appData: ApplicationData) => {
-  if (moPrivateData !== undefined && moPrivateData.length > 0) {
```

Comment:
We don't need this line as its checked before the mapManagingOfficersPrivateData method call.